README.md

```diff
@@ -37,6 +37,8 @@ In the project properties for **pdf-java-toolkit-samples**, select the **Maven**
 
 It will ask if you want to update the product configuration. Click **Yes**.
 
+The samples were developed and tested using [Mars](https://eclipse.org/mars/)
```

Comment:
@datalogics-tsmith, if we said something like Mars 4.5.0, I would be happy as it seems like that is the earliest that any of us are using.